src/ripple/ledger/impl/View.cpp

```diff
@@ -44,6 +44,19 @@ bool amendmentRIPD1141 (NetClock::time_point const closeTime)
     return closeTime > amendmentRIPD1141SoTime();
 }
 
+NetClock::time_point const& amendmentRIPD1274SoTime ()
+{
+    using namespace std::chrono_literals;
+    // Thu Sept 8, 2016 10:00:00am PDT
+    static NetClock::time_point const soTime{526669200s};
```

Comment:
I believe the point of this switchover time is so all validators start applying the new rules at the same time.  If that's correct, do we believe this code will be on all validators by the morning of Sept 8?  Just wondering.